src/components/SpectroscopyForm.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from "react";
+import { Box, TextField, Typography } from "@mui/material";
+import NumberTextField from "./NumberTextField";
+import RunPlanButton from "./RunPlanButton";
+
+export type SpectroscopyFormData = {
+  numbers_of_points: number;
+  grid_size: number;
+  grid_origin_x: number;
+  grid_origin_y: number;
+  exposure_time: number;
+};
+
+function SpectroscopyForm() {
+  const [formData, setFormData] = useState<SpectroscopyFormData>({
+    numbers_of_points: 5,
+    grid_size: 5.0,
+    grid_origin_x: 0.0,
+    grid_origin_y: 0.0,
+    exposure_time: 0.1,
+  });
+  const [instrumentSession, setInstrumentSession] = useState("");
+
+  return (
+    <Box
+      sx={{
+        margin: 2,
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <Typography
+        variant="h5"
+        component="h1"
+        sx={{ mb: 2, fontWeight: "bold" }}
+      >
+        spectroscopy_scan
+      </Typography>
+      <Box
+        sx={{
+          display: "grid",
+          gridTemplateColumns: {
+            xs: "1fr",
+            sm: "1fr 1fr",
+            md: "1fr 1fr 1fr",
+          },
+          gap: 3,
+          flexGrow: 1,
+        }}
+      >
+        <NumberTextField
```

Comment:
Having origin x top right and origin bottom left (as it shows now) feels disorganised to me. This is minor because talking to Tom Cobb, this form should get even simpler, but if you wanted to organise these fields in threes I'd do X start, Y start, Size; Points, Exposure, Instrument session (which eventually wouldn't be a UI parameter either).